recipes/lz4/all/conanfile.py

```diff
@@ -71,4 +74,4 @@ def package_info(self):
         self.cpp_info.names["pkg_config"] = "liblz4"
         self.cpp_info.libs = ["lz4"]
         if self.settings.compiler == "Visual Studio" and self.options.shared:
-            self.cpp_info.defines.append("LZ4_DLL_IMPORT")
+            self.cpp_info.defines.append("LZ4_DLL_IMPORT=1")
```

Comment:
as I remember, `-DLZ4_DLL_IMPORT` is equivalent to `-DLZ4_DLL_IMPORT=1`, is there an actual fix? what exactly was an issue?

Reply:
https://github.com/lz4/lz4/blob/db6edc43f78460ca2dc4b74b6e0f402cdf2e65d5/lib/lz4.h#L94

Reply:
Actually, even if LZ4_DLL_IMPORT was not defined at consume time, it would work because `__declspec(dllimport)` is an hint (except for global variables).

Reply:
the code checks that `LZ4_DLL_IMPORT` is defined, and that `LZ4_DLL_IMPORT` has value 1.
however, from GCC [documentation](https://gcc.gnu.org/onlinedocs/gcc/Preprocessor-Options.html):
```
-D name
Predefine name as a macro, with definition 1.
```
and Visual C++ [documentation](https://docs.microsoft.com/en-us/cpp/build/reference/d-preprocessor-definitions?view=msvc-160):
```
By default, the value associated with a symbol is 1. For example, /D name is equivalent to /D name=1. In the example at the end of this article, the definition of TEST is shown to print 1.
```

therefore, I believe, the following should have the same effect for all compilers:
```
            self.cpp_info.defines.append("LZ4_DLL_IMPORT")
            self.cpp_info.defines.append("LZ4_DLL_IMPORT=1")
```

did you see `self.cpp_info.defines.append("LZ4_DLL_IMPORT")` resulting in different value?

Reply:
Ok I see, no I didn't check, but you're probably right.